.github/patch-types.php

```diff
@@ -25,6 +25,7 @@
         case false !== strpos($file = realpath($file), '/vendor/'):
         case false !== strpos($file, '/src/Symfony/Bridge/PhpUnit/'):
         case false !== strpos($file, '/src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Validation/Article.php'):
+        case false !== strpos($file, '/src/Symfony/Component/Cache/Tests/Fixtures/DriverWrapper.php'):
```

Comment:
why that? can't you add the return types to the file instead?

Reply:
The `path return types` runs with DBAL v2 installed, whereas this `DriverWrapper` class implements the interface of v3. Tested it again to be sure with return types: https://github.com/symfony/symfony/pull/42011/checks?check_run_id=3054146971

Reply:
Since I couldn't make that work (it would mean making the `DriverWrapper` compatible with both v2 and v3), and the `DriverWrapper` is only loaded for the dbal v3 tests, I've re-added this exclusion. I hope that's okay.